lib-api/src/main/java/org/ergoplatform/appkit/babelfee/BabelFeeBoxState.java

```diff
@@ -0,0 +1,144 @@
+package org.ergoplatform.appkit.babelfee;
+
+import org.ergoplatform.appkit.ErgoId;
+import org.ergoplatform.appkit.ErgoToken;
+import org.ergoplatform.appkit.ErgoValue;
+import org.ergoplatform.appkit.InputBox;
+import org.ergoplatform.appkit.OutBox;
+import org.ergoplatform.appkit.OutBoxBuilder;
+import org.ergoplatform.appkit.Parameters;
+import org.ergoplatform.appkit.SigmaProp;
+import org.ergoplatform.appkit.TransactionBox;
+import org.ergoplatform.appkit.UnsignedTransactionBuilder;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+/**
+ * Represents a Babel Fee Box state, see EIP-0031
```

Comment:
I suggest to copy the definition of Babel Fee Box state here from the EIP and maybe provide additional useful description here. When reading the code it is often enough to get general idea and it is really annoying to always go to specification. 
Also please put URL link to the spec.

Reply:
Done, but URL is not yet valid (EIP is not merged)

Reply:
You can use stable url of the file beloging to specific commit.

Reply:
done